### PR TITLE
JENKINS-74732 - Move jelly inline event handlers and script blocks to JavaScript. 

### PR DESCRIPTION
**Pull request by asimell**
* Moved all `onclick` handlers and `script` blocks to `robot.js`
* Renamed `.expand` and `.collapse` classes to specifically pinpoint this plugin to avoid any possible errors with other plugins.

### CODE REVIEW

File: src/main/webapp/robot.js [hunk 1]
```diff
@@ -112,3 +114,40 @@ function hideStackTrace(id) {
...
+            if (url == "")
+                document.getElementById("maxBuildsToShow").addEventListener("change", redrawDurationGraph);
+        }
+        initPassFailGraph(url);
+        if (url == "")
+            initDurationGraph(url);
+    }
+});
+
+document.addEventListener("DOMContentLoaded", function(){
```
Comment (Tattoo) [inline src/main/webapp/robot.js:139]:
```suggestion

```

Comment (Tattoo) [reply src/main/webapp/robot.js:139]: I don't think you can define two different functions to `DOMContentLoaded` without overwriting the former with the latter. These should probably be combined together as a single startup

**Review by Tattoo — COMMENTED** [review]
Otherwise, LGTM

File: src/main/webapp/robot.js (hunk 1 above)
Comment (asimell) [reply src/main/webapp/robot.js:139]: Worked on my machine, but I'll refactor this. It probably makes more sense to have them combined anyway.

**Review by Tattoo — APPROVED** [review]
💖

**Pull request merged**